app/views/family_members/index.html.erb

```diff
@@ -1,5 +1,7 @@
 <p style="color: green"><%= notice %></p>
 
+<button><%= link_to "Home", pages_path, class: "btn btn-primary" %></button>
```

Comment:
move repeating code to a reusable file, like partial

Reply:
repeating code is moved to partial file.